sample.tf

```diff
@@ -10,10 +10,10 @@ terraform {
 
 provider "aws" {
   profile = "default"
-  region  = "us-west-2"
+  region  = "us-west-1"
 }
 
-resource "aws_instance" "app_server" {
+resource "aws_instance" "app_seraver" {
```

Comment:
Issue - Ensure that your AWS application is not deployed within the default Virtual Private Cloud in order to follow security best practices 
 category - Infrastructure Security 
 severity - MEDIUM